contractcourt/chain_arbitrator.go

```diff
@@ -891,7 +891,7 @@ func (c *ChainArbitrator) Stop() error {
 	}
 	for chanPoint, arbitrator := range activeChannels {
 		log.Tracef("Attempting to stop ChannelArbitrator(%v)",
-			chanPoint)
+			arbitrator.id())
```

Comment:
IMO, the scid is _much_ less useful than the chan point. For the chan point we can easily use it to look up the transaction/channel. This is frequently done in incoming user bug reports. If you only have the `scid`, you need to translate it to get the outpoint, which is what's commonly used.

Reply:
Got it, changed to chan point instead.